### ISSUE TITLE
no-dynamic-styling rule should report the computed value

### ISSUE THREAD

**Issue by nusuke** [chakra-ui/eslint-plugin-panda#136]
When I write the following `cva` variants, `@pandacss/no-dynamic-styling` do not reports an error.

```tsx
import { computedValue } from "./const";
const heading = cva({
  base: {
    fontWeight: "strong",
    lineHeight: "m",
  },
  variants: {
    [computedValue]: {
      color: "red.100",
    },
  },
});
```

If a variable imported from an external file is passed to the cva function as a computedValue, panda will not work correctly.
So I want to make an error with @pandacss/no-dynamic-styling in lint.

https://github.com/chakra-ui/eslint-plugin-panda/blob/main/docs/rules/no-dynamic-styling.md

## Reproduce Repository

https://github.com/nusuke/next-panda-sandbox/blob/main/src/app/_components/Heading.tsx#L22-L34

## Actual

@pandacss/no-dynamic-error No Error

## Expected

Error.

The computedValue is a dynamic value; I want to make an error in lint so that I cannot pass a dynamic value to RecipeVariant.
